libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -51,11 +51,8 @@
 
 
  // console default baud rates and buffer sizes
-#ifdef HAL_SERIAL0_BAUD_DEFAULT
-# define AP_SERIALMANAGER_CONSOLE_BAUD          HAL_SERIAL0_BAUD_DEFAULT
-#else
+
 # define AP_SERIALMANAGER_CONSOLE_BAUD          115200
```

Comment:
can we remove the CONSOLE defines now?

Reply:
it is used in AP_Periph and ESP32 also, as well as the default for serial 0